src/test/java/hw3/pages/components/PageIndexComponent.java

```diff
@@ -0,0 +1,44 @@
+package hw3.pages.components;
+
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+public class PageIndexComponent extends AbstractComponent {
+
+    public PageIndexComponent(WebDriver driver) {
+        super(driver);
+    }
+
+    @FindBy(className = "benefit-icon")
+    List<WebElement> imageWebElementList;
+
+    @FindBy(className = "benefit-txt")
+    List<WebElement> textWebElementList;
+
+
+    public int getImageWebElementListSize() {
+        return this.imageWebElementList.size();
+    }
```

Comment:
What is the order for **fields**, **constructors**, **methods** for class file described in Java Code Convention?